src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export function Person({ person }) {
+  let partnerInfo;
+
+  if (!person.isMarried) {
+    partnerInfo = 'I am not married';
+  } else if (person.sex === 'm') {
+    partnerInfo = `${person.partnerName} is my wife`;
+  } else {
+    partnerInfo = `${person.partnerName} is my husband`;
+  }
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
+      {person.age !== undefined && (
+        <p className="Person__age">I am {person.age}</p>
+      )}
```

Comment:
Fix it after destructuring
```suggestion
      {age && <p className="Person__age">I am {age}</p>}
```